tests/foreman/cli/test_activationkey.py

```diff
@@ -1001,7 +984,7 @@ def test_bugzilla_1111723(self):
         self.assertEqual(new_activation_key['name'], name)
 
     @tier2
-    def test_remove_hc_by_id(self):
+    def test_positive_add_host_collection_by_id(self):
```

Comment:
Docstrings says `remove`. Should this be `test_positive_remove_host_collection_by_id`?